### PR TITLE
adding new chain BILLIONAIRE CLUB mainnet 

### PR DESCRIPTION
**Pull request by bcc2025**
did all build & formatting process

### CODE REVIEW

**Comment by bcc2025** [comment]
@ligi Hi, Could you please take a moment to review this PR? Getting our chain on Chainlist is time-sensitive. Thanks in advance!

**Review by Blyla — APPROVED** [review]

**Comment by bcc2025** [comment]
@Blyla Hi, Could you please take a moment to review this PR? Getting our chain on Chainlist is time-sensitive. Thanks in advance!

**Comment by ligi** [comment]
If it is time-sensitive please pay funds to lists.eth (in your case 0.5ETH) - otherwise this PR will sit for a while

File: _data/icons/billionaireclub.json
Comment (Jpazm01) [inline _data/icons/billionaireclub.json:1]: Who what do I do here?

**Review by ligi — CHANGES_REQUESTED** [review]
image

**Pull request closed without merging**